rest-tests/src/test/resources/features/integration-lifecycle.feature

```diff
@@ -0,0 +1,56 @@
+Feature: integration lifecycle scenarios
+
+  Background: Create sample integration
+    Given clean application state
+    And clean all builds
+    And create DB step with query: "SELECT * FROM TODO" and interval: 4000 miliseconds
+    And create DB insert taks step
+
+  @integrations-lifecycle
+  Scenario: lifecycle integration draft
+    When create new integration with name: "DB to DB rest test draft" and desiredState: "Draft"
+    Then verify there are no s2i builds running for integration: "DB to DB rest test draft"
+
+  @integrations-lifecycle @integrations-lifecycle-activate
+  Scenario: lifecycle integration activate
+    When create new integration with name: "DB to DB rest test activate" and desiredState: "Draft"
+    Then set integration with name: "DB to DB rest test activate" to desiredState: "Active"
+    Then wait for integration with name: "DB to DB rest test activate" to become active
+
+  @integrations-lifecycle @integrations-lifecycle-act-deact
+  Scenario: lifecycle integration act-deact
+    When create new integration with name: "DB to DB rest test act-deact" and desiredState: "Active"
+    Then wait for integration with name: "DB to DB rest test act-deact" to become active
+    Then set integration with name: "DB to DB rest test act-deact" to desiredState: "Inactive"
+    Then validate integration: "DB to DB rest test act-deact" pod scaled to 0
+    Then set integration with name: "DB to DB rest test act-deact" to desiredState: "Active"
+    Then validate integration: "DB to DB rest test act-deact" pod scaled to 1
+
+  @integrations-lifecycle @integrations-lifecycle-try-with-same-name
+  Scenario: lifecycle integration act-deact
+    When create new integration with name: "DB to DB rest same name" and desiredState: "Active"
+	Then wait for integration with name: "DB to DB rest same name" to become active
+    Given create DB step with query: "SELECT * FROM TODO" and interval: 4000 miliseconds
+    And create DB insert taks step
+    Then try to create new integration with the same name: "DB to DB rest same name" and state: "Draft"
+
+  @integrations-lifecycle @integrations-lifecycle-long
+  Scenario: lifecycle integration act-deact 5 times
+    When create new integration with name: "DB to DB rest test act-deact" and desiredState: "Active"
```

Comment:
This should be combination of 
```
When set...
The validate...
When set...
...
```

Reply:
Going through the second time. I've realized that the scenario could be implemented with a meta step that would encapsulate switching within a for-loop and take a param how many times. Wdyt?

Reply:
ok, will do

Reply:
Updated, have a look please.